src/parameterized_tendencies/microphysics/precipitation.jl

```diff
@@ -27,9 +27,9 @@ precipitation_tendency!(Yₜ, Y, p, t, colidx, ::NoPrecipitation) = nothing
 function precipitation_cache(Y, precip_model::Microphysics0Moment)
     FT = Spaces.undertype(axes(Y.c))
     return (;
-        ᶜS_ρq_tot = similar(Y.c, FT),
-        ᶜ3d_rain = similar(Y.c, FT),
-        ᶜ3d_snow = similar(Y.c, FT),
+        ᶜS_ρq_tot = zeros(axes(Y.c)),
+        ᶜ3d_rain = zeros(axes(Y.c)),
+        ᶜ3d_snow = zeros(axes(Y.c)),
```

Comment:
Is it causing problems if these are initialized using `similar`? For some reason I don't fully understand, I think it is preferred to not have prescribed initial values for the cache, if not needed.

Reply:
Let me try similar with the final version...